modules/geolocation/geolocationWorker.py

```diff
@@ -61,8 +61,8 @@ def geolocation_locator_worker(pause, exitRequest, pipelineIn, pipelineOut, pipe
     return
 
 
-def geolocation_output_worker(pause, exitRequest, pipelineIn, pipelineOut, pipelineInLock):
-
+def geolocation_output_worker(pause, exitRequest, pipelineIn, pipelineInLock):
+    # No pipelineOut queue, instead writes locations to CSV file
```

Comment:
Use the triple quotation marks style for the comment (this will allow IDEs to display the comment on inspection).

```py
"""My comment OR"""
'''My comment'''
```